calibrators/docs/tutorials/marker_radar_lidar_calibrator.md

```diff
@@ -0,0 +1,237 @@
+# marker_radar_lidar_calibrator
+
+In this tutorial, we present a hands-on tutorial of the `marker_radar_lidar_calibrator`. Although we provide a pre-recorded rosbag, the flow of the tutorial is meant to show the user the steps they must perform in their own use cases with live sensors.
+
+General documentation regarding this calibrator can be found [here](../../marker_radar_lidar_calibrator/README.md).
+
+## Setup
+
+This tutorial assumes that the user has already built the calibration tools.
+Installation instructions can be found [here](../../README.md).
+
+## Data preparation
+
+Please download the data (rosbag) from [here](https://drive.google.com/drive/folders/1G1n2AV47gbQFKQtlzbHJPeV10m5_uMMk).
+
+The rosabg includes three different topics: `object_raw`, `pointcloud_raw`, and `tf_static`.
```

Comment:
This does not seem to be correct. Maybe this applies to other documents

```
Topic information: Topic: /sensing/lidar/front_lower/pointcloud_raw | Type: sensor_msgs/msg/PointCloud2 | Count: 2064 | Serialization Format: cdr
                   Topic: /sensing/radar/front_center/objects_raw | Type: radar_msgs/msg/RadarTracks | Count: 2738 | Serialization Format: cdr
                   Topic: /tf_static | Type: tf2_msgs/msg/TFMessage | Count: 1 | Serialization Format: cdr

```

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/76db984ad1a6e8dab27ee9e0f9ed7b0dbb84e55a

Reply:
That was not the problem. Please read the standards and limitations for ros topic names if you do not understand what I mean

Reply:
Do you mean I should put the "/sensing/lidar/front_lower/" in front?

Reply:
Yes

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/beb89716dd55c214a73877e67fbfc081a72ac865

Reply:
fix the `tf_static` to ` /tf_static` in https://github.com/tier4/CalibrationTools/pull/164/commits/b16200b7fed0e3b9fc4da134111a92ff42a6c8db